src/NServiceBus.Core/Features/FeatureActivator.cs

```diff
@@ -225,14 +221,24 @@ public FeatureInfo(Feature feature, FeatureDiagnosticData diagnostics)
                 Feature = feature;
             }
 
+            public void AddTaskControllers(IList<FeatureStartupTaskController> controllersToAdd)
+            {
+                foreach (var taskController in controllersToAdd)
```

Comment:
Maybe use `AddRange`? With AddRange it would also be questionable whether you really need a method to add to the list